From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Tue, 10 Nov 2015 22:13:39 +0100
Subject: Don't export catkin includes into tf2*Config.cmake

They only include the temporary include in the build directory.
---
 tf2_bullet/CMakeLists.txt | 2 +-
 tf2_kdl/CMakeLists.txt    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tf2_bullet/CMakeLists.txt b/tf2_bullet/CMakeLists.txt
index dd4cc04..3b10ea2 100644
--- a/tf2_bullet/CMakeLists.txt
+++ b/tf2_bullet/CMakeLists.txt
@@ -10,7 +10,7 @@ find_package(catkin REQUIRED COMPONENTS geometry_msgs tf2)
 
 include_directories(include ${bullet_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
 
-catkin_package(INCLUDE_DIRS include ${bullet_INCLUDE_DIRS} ${catkin_INCLUDE_DIRS})
+catkin_package(INCLUDE_DIRS include ${bullet_INCLUDE_DIRS})
 
 install(DIRECTORY include/${PROJECT_NAME}/
         DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
diff --git a/tf2_kdl/CMakeLists.txt b/tf2_kdl/CMakeLists.txt
index 1e35e1f..ca1415f 100644
--- a/tf2_kdl/CMakeLists.txt
+++ b/tf2_kdl/CMakeLists.txt
@@ -7,7 +7,7 @@ find_package(catkin REQUIRED COMPONENTS cmake_modules tf2 tf2_ros tf2_msgs)
 find_package(Eigen REQUIRED)
 
 catkin_package(
-  INCLUDE_DIRS include ${catkin_INCLUDE_DIRS} ${Eigen_INCLUDE_DIRS}
+  INCLUDE_DIRS include ${Eigen_INCLUDE_DIRS}
   DEPENDS Eigen orocos_kdl
 )
 
